dev/TabView/TabView_v1.xaml

```diff
@@ -142,6 +142,7 @@
                         contract7NotPresent:CornerRadius="{ThemeResource ControlCornerRadius}">
                         <ScrollViewer x:Name="ScrollViewer"
                             Grid.Column="1"
+                            Padding="0,0,-1,0"
```

Comment:
Why do we need this negative padding all of the sudden? Negative paddings seem very weird here...

Reply:
It cancels the width of the rightmost separator, which is not customizable yet, so it's set as -1px.

Reply:
But shouldn't the right most converter be collapsed?

Reply:
I just checked again and that padding actually doesn't do anything. It was probably for a different scrapped PR, so I'll remove it.